src/equations/equations.jl

```diff
@@ -61,7 +61,7 @@ Given the conservative variables `u`, calculate the (physical) flux in spatial
 direction `orientation` for the coressponding set of governing `equations`
 `orientation` is `1`, `2`, and `3` for the x-, y-, and z-directions, respectively.
 """
-function flux(u, orientation, equations) end
+function flux end
```

Comment:
Why this change? Does not seem to be related to the rest of the PR, or am I missing something? When changing this, one should probably also change the docstring.

Reply:
That defined a general function `flux(_, _, _) = nothing`, e.g. `flux(1, 2, 3)`. I just noticed this while working on this PR and fixed the way how to write a docstring for a function without defining a method for it at the same time.

Reply:
Thanks for the explanation!